release/platform/platform.go

```diff
@@ -367,4 +367,12 @@ var platforms = []Platform{
 		Repos:     []string{RepoOrg, RepoEnterprise},
 		BuildTags: []string{"ssl", "sasl", "gssapi", "failpoints"},
 	},
+	{
+		Name:      "rhel-82",
+		Arch:      "arm64",
```

Comment:
a couple suggestions:
- let's add a `RPMArch()` method similar to `DebianArch()`
- let's create constants for these architecture strings, similar to what we've done for OS, Pkg, and Repo